features/FEATURE_BLE/ble/gap/AdvertisingParameters.h

```diff
@@ -402,8 +400,12 @@ class AdvertisingParameters {
      */
     AdvertisingParameters &setFilter(advertising_filter_policy_t mode)
     {
+#if BLE_FEATURE_WHITELIST
         _policy = mode;
         return *this;
+#else
+#error "Whitelist feature disabled"
```

Comment:
So this will trigger an error whenever whitelist is disabled ?

Reply:
yes, that is wrong